<launch>
  <node pkg="ros_network_gap" type="ros_network_gap.py" name="network_gap_hostname" output="screen">
  	<param name="broadcast" value="10.10.50.255" /> 	<!-- broadcast addr of network-->
	<param name="port" value="14317" />  			<!-- UDP listener port -->
	<param name="msg_pkg" value="nav_msgs.msg" />  		<!-- package name of message + .msg for python namespace -->
	<param name="msg_type" value="Odometry" />		<!-- Message type -->
	<param name="max_rate" value="5" />			<!-- Maximum rate of publishing to other networks -->
	<remap from="network_gap_topic" to="odometry_exchange" />
  </node>
</launch>
